modules/client-common/src/context-core.ts

```diff
@@ -3,48 +3,39 @@ import { Client as IpfsClient } from "@aragon/sdk-ipfs";
 import { GraphQLClient } from "graphql-request";
 import { isAddress } from "@ethersproject/address";
 import { Signer } from "@ethersproject/abstract-signer";
+import { ContextParams, ContextState, OverriddenState } from "./types";
 import {
-  ContextParams,
-  ContextState,
-  OverriddenState,
-  SupportedNetwork,
-  SupportedNetworksArray,
-  SupportedVersion,
-} from "./types";
-import { GRAPHQL_NODES, IPFS_NODES, LIVE_CONTRACTS } from "./constants";
-import { getNetwork } from "./utils";
-import { DeployedAddressesArray } from "./internal";
+  getDefaultGraphqlNodes,
+  getDefaultIpfsNodes,
+  getNetwork,
+} from "./utils";
 import {
   InvalidAddressError,
   InvalidGasEstimationFactorError,
   UnsupportedNetworkError,
   UnsupportedProtocolError,
 } from "./errors";
+import {
+  ContractNames,
+  getNetworkDeploymentForVersion,
+  getNetworkNameByAlias,
+  SupportedVersions,
+} from "@aragon/osx-commons-configs";
 
 const DEFAULT_GAS_FEE_ESTIMATION_FACTOR = 0.625;
 const supportedProtocols = ["https:"];
+const contractNames = Object.values(ContractNames);
 if (typeof process !== "undefined" && process?.env?.TESTING) {
   supportedProtocols.push("http:");
 }
 
 export abstract class ContextCore {
   protected state: ContextState = {} as ContextState;
-  protected overriden: OverriddenState = {
-    daoFactoryAddress: false,
-    pluginSetupProcessorAddress: false,
-    multisigRepoAddress: false,
-    adminRepoAddress: false,
-    addresslistVotingRepoAddress: false,
-    tokenVotingRepoAddress: false,
-    multisigSetupAddress: false,
-    adminSetupAddress: false,
-    addresslistVotingSetupAddress: false,
-    tokenVotingSetupAddress: false,
-    ensRegistryAddress: false,
-    gasFeeEstimationFactor: false,
-    ipfsNodes: false,
-    graphqlNodes: false,
-  };
+
+  protected overriden: OverriddenState = contractNames.reduce(
+    (acc, key) => ({ ...acc, [key]: false }),
+    { ENSRegistry: false } as OverriddenState,
```

Comment:
Non blocker but in general I'd avoid the use of reduce statements like this. The before example is much clearer to future readers of your code.